README.md

```diff
@@ -28,52 +28,70 @@ Before you proceed, make sure you have the following software installed:
     On Debian/Ubuntu:
     
     ```bash
-    $ sudo apt install libpq-dev
+     sudo apt install libpq-dev
     ```
+ * Cargo dependency
+ * 
+   On Debian/Ubuntu:
+    
+    ```bash
+     sudo apt install cargo
+    ```
+    
+### Prepare Database
 
+Setup PostgreSQL database
 
-### Prepare Database
+ ```bash
+ sudo apt-get install postgresql postgresql-contrib
+```   
 
-Setup PostgreSQL database, create a database with the regular tools, and note the connection string (database host, credentials, and the database name).
+Create a database with the regular tools (Change user and db_name with yours)
+    ```bash
+     sudo su postgres
+     psql
+     create database db_name with owner=user;
+    ```
+Note the connection string (database host, credentials, and the database name).
```

Comment:
Meh, setting up a database is a complicated topic, so I would avoid giving any instructions. Consider finding a good article out there and link it from this README

Reply:
Or we can just give it as a example. as you want